cmd/sdkch/README.md

```diff
@@ -0,0 +1,174 @@
+# sdkch
+Simple tool to maintain modular changelogs
+
+# Usage
+
+```
+$ sdkch -help
+usage: sdkch COMMAND
+
+Maintain unreleased changelog entries in a modular fashion.
+
+Commands:
+    init
+    add SECTION STANZA            Add an entry file.
+                                  Read from stdin until it
+                                  encounters EOF.
+    generate [VERSION]            Generate a changelog in
+                                  Markdown format and print it
+                                  to stdout. VERSION defaults
+                                  to UNRELEASED.
+
+    Sections             Stanzas
+         ---                 ---
+    breaking                gaia
+    features             gaiacli
+improvements            gaiarest
+    bugfixes                 sdk
+                      tendermint
+
+Flags:
+  -d string
+    	entry files directory (default "./.pending")
+```
+
+## Initialise a new entry files directory
+
+First, you need to initialise a new directory that will host entry files:
+
+```bash
+$ sdkch init
+```
+
+A new hidden directory `.pending` is created. Its 2-level structure looks like the following:
+
+```
+$ tree .pending/
+.pending/
+|-- breaking
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- bugfixes
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- features
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+`-- improvements
+    |-- gaia
+    |-- gaiacli
+    |-- gaiarest
+    |-- sdk
+    `-- tendermint
+
+24 directories, 0 files
+```
+
+Deepest level directories can contain `entry files`. Entry files are created before submitting a PR for review.
+
+## Add a new entry
+
+You can either drop a text file in the appropriate directory or use the `add` command:
+
+```bash
+$ sdkch add features sdk
+[\#3813] New sdk.NewCoins safe constructor to replace bare
+sdk.Coins{} declarations.
+Written to .pending/features/sdk/3813-New-sdk-NewCoins-safe-constructor-to-replace-bare
+```
+
+`add` reads a description from STDIN, creates a new file and drop it in the appropriate
+folder according to the supplied arguments.
+
+## Generate the full changelog
+
+```bash
+find .pending -type f
+```
+
+```bash
+$ sdkch generate
```

Comment:
During this process (or alternatively automatically during the add process) links should be added in to any numbers provided like `\#2345` 
aka this process should convert `\#2345` -> `[\#2345](https://github.com/cosmos/cosmos-sdk/issues/2345)`

Reply:
We have a separate tool for that: `linkify`

Reply:
...but i've done it anyway